### ISSUE TITLE
required dependencies are pointing to older versions [Karma, slack, shelljs, uuid]

### ISSUE THREAD

**Issue by AmirHussain93** [IBMa/equal-access#740]
I have updated module to latest **v3.1.22**, but still required dependencies like karma, slack,shelljs, uuid are pointing to their older versions which have security vulnerability issues. Please refer the screenshot below

<img width="945" alt="image" src="https://user-images.githubusercontent.com/27803197/158570365-e9e750bb-8084-44d1-8bcd-b46e5e803ec8.png">

karma 4.4.1 **(latest v6.3.17)**
slack 8.1.1 **(latest v11.0.2)**
shelljs 0.7.5 **(latest v0.8.5)**
uuid 2.0.1 **(latest v8.3.2)**

Could you please update the above packages to latest?  We are getting security vulnerability alerts in git which are critical due to this.
